src/libs/Environment/Environment.js

```diff
@@ -31,8 +37,21 @@ function getEnvironmentURL() {
     });
 }
 
+/**
+ * Get the corresponding oldDot URL based on the environment we are in
+ *
+ * @returns {Promise}
```

Comment:
```suggestion
 * @returns {Promise<String>}
```

Returns a Promise that resolves to a string